blues/moves.py

```diff
@@ -0,0 +1,256 @@
+"""
+moves.py: Provides the two main classes, Move and MoveEngine
+which allow altering the positions of a
+subset of atoms in a context during a BLUES simulation to
+increase sampling.
+
+Authors: Samuel C. Gill
+Contributors: Nathan M. Lim, David L. Mobley
+"""
+
+import parmed
+from simtk import unit
+import mdtraj
+import numpy as np
+import sys, traceback
+
+
+class Move(object):
+
+    """Model provides methods for calculating properties on the
```

Comment:
This doc string belongs to `Model`, not to `Move`, and needs updating, @sgill2 .